__tests__/puppeteer_test.re

```diff
@@ -673,7 +673,7 @@ describe("Coverage", () => {
            |> Coverage.startJSCoverage(_, Coverage.makeJSCoverageOptions())
            |> then_(() => page |> Page.goto("file://" ++ testPagePath, ()))
            |> then_(_res => coverage |> Coverage.stopJSCoverage)
-           |> then_(report => report |> expect |> toMatchSnapshot |> resolve);
+           |> then_(report => report |> expect |> toHaveLength(1) |> resolve);
```

Comment:
I got error from: https://travis-ci.org/bs-puppeteer/bs-puppeteer/builds/362974031#L1805

```
  ● Coverage › startJSCoverage, stopJSCoverage
    expect(value).toMatchSnapshot()
    
    New snapshot was not written. The update flag must be explicitly passed to write a new snapshot.
    
    This is likely because this test is run in a continuous integration (CI) environment in which snapshots are not written by default.
    
    Received value 
    Array [
      Object {
        "ranges": Array [
          Object {
            "end": 19,
            "start": 0,
          },
          Object {
            "end": 63,
            "start": 37,
          },
        ],
        "text": "
    		function foo() {function bar() { } console.log(1); } foo(); ",
        "url": "file:///home/travis/build/bs-puppeteer/bs-puppeteer/__tests__/fixtures/testPage.html",
      },
    ]
      
      
      at affirm (node_modules/@glennsl/bs-jest/lib/js/src/jest.js:184:35)
      at node_modules/@glennsl/bs-jest/lib/js/src/jest.js:269:48
```

I think the major issue here is `url` in snapshot, so I just simply change to check the length.

Reply:
Ah I see. Well we can at least add a couple of the same checks the snapshot would give us without using snapshots. We can check that the url contains `fixtures/testPage.html`, and we can check that the ranges are right.

Reply:
Okay. Fixed in commit 583a700